spec/services/conference_sort_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Decidim::Conferences
+  describe ConferenceSort do
+    let(:organization) { create(:organization) }
+    let(:upcoming_conference_one) { create(:conference, :published, promoted: false, organization: organization, start_date: 1.week.from_now, end_date: 2.weeks.from_now) }
+    let(:upcoming_conference_two) { create(:conference, :published, promoted: false, organization: organization, start_date: 2.weeks.from_now, end_date: 3.weeks.from_now) }
+    let(:past_conference_one) { create(:conference, :published, promoted: false, organization: organization, start_date: 2.months.ago, end_date: 1.month.ago) }
+    let(:past_conference_two) { create(:conference, :published, promoted: false, organization: organization, start_date: 3.months.ago, end_date: 2.months.ago) }
+
```

Comment:
Maybe we should add more cases ordered in different ways to be sure that are not created in the database in the same order that will be checked after.